en/SettingsView/MAVLink.md

```diff
@@ -5,26 +5,25 @@ This includes setting the MAVLink system ID for *QGroundControl* and viewing lin
 
 The screen also allows you to manage [MAVLink 2 Log Streaming](#logging) (PX4 only), including *automating log upload to Flight Review*!
 
-![MAVLink settings screen](../../assets/settings/mavlink/overview.jpg)
+![MAVLink settings screen](../../assets/settings/mavlink/overview.png)
 
 ## Ground Station {#ground_station}
 
 This section sets the MAVLink properties and behaviour of *QGroundControl*.
 
-![Ground Station](../../assets/settings/mavlink/ground_station.jpg)
+![Ground Station](../../assets/settings/mavlink/ground_station.png)
 
 - **MAVLink System ID:** System ID of *QGroundControl* on the MAVLink network (Default: 255).
-  Vehicles are typically allocated IDs from 1. 
-  You may have to specify another ID if there are multiple ground stations or MAVLink applications on the network. 
+  Vehicles are typically allocated IDs from 1.
+  You may have to specify another ID if there are multiple ground stations or MAVLink applications on the network.
 - **Emit heartbeat:** Disable emission of regular MAVLink HEARTBEAT message (Default: True).
   Generally speaking, you should not turn this off.
-- **Request start of MAVLink telemetry streams (ArduPilot only):** Enable *QGroundControl* requesting the standard-rate ArduPilot telemetry streams (Default: True). 
```

Comment:
Did you try this on ArduPilot? i.e. This might have been removed, or it might now be conditionally displayed.

Reply:
Actually, these are App Settings, so should not be vehicle specific (although someone could have made them such). I will assume not. Ignore. Thanks.